package.yaml

```diff
@@ -22,27 +22,36 @@ description:         Please see the README on GitHub at <https://github.com/gith
 
 dependencies:
```

Comment:
In practice, can we add these as test dependencies, or do we need them in the app?

Reply:
Yeah, you can have them as test-only dependencies.